configuration/white_labels/ma.py

```
@@ -327,28 +327,30 @@ def get_white_label(self) -> WhiteLabel:
-            "_default_message": "Homeowners Insurance",
+        "dependentCare": {
+            "childCare": {"_label": "expenseOptions.childCare", "_default_message": "Child Care"},
+            "childSupport": {"_label": "expenseOptions.childSupport", "_default_message": "Child Support (Paid)"},
+            "dependentCare": {"_label": "expenseOptions.dependentCare", "_default_message": "Dependent Care"},
         },
-        "medical": {"_label": "expenseOptions.medical", "_default_message": "Medical Insurance Premium &/or Bills"},
-        "cooling": {"_label": "expenseOptions.cooling", "_default_message": "Cooling"},
-        "childCare": {"_label": "expenseOptions.childCare", "_default_message": "Child Care"},
-        "childSupport": {"_label": "expenseOptions.childSupport", "_default_message": "Child Support (Paid)"},
-        "dependentCare": {"_label": "expenseOptions.dependentCare", "_default_message": "Dependent Care"},
     }
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)